src/components/VideoPlayer/VideoPlayer.component.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEye, faThumbsDown, faThumbsUp } from '@fortawesome/free-solid-svg-icons';
+import { useVideo } from '../../providers/Video';
+import useVideoPlayerApi from '../../hooks/useVideoPlayerApi';
+import VideoSuggestions from '../VideoSuggestions';
+import {
+  VideoPlayerContainer,
+  Video,
+  AsideSuggestions,
+  Info,
+  VideoDescription,
+  VideoStatistics,
+  Tags,
+  Tag,
+  VideoRating,
+} from './VideoPlayer.styles';
+import VideoWrapper from '../VideoWrapper/VideoWrapper.component';
+
+function VideoPlayer({ videoId }) {
+  const { setAlert } = useVideo();
+  const { video, isLoading, error } = useVideoPlayerApi({
+    type: 'VIDEO_DETAILS',
+    payload: videoId,
+  });
```

Comment:
This looks great, you mentioned that you are working on the search to use context, which is something for required for the next challenge, you could use something like this hook for the search request

Reply:
of course, that was my idea use it on `useVideoPlayerApi ` for the search and video details page. but the response (returned values) were slightly different for one i need a single video and for search a list